src/main/java/frc/robot/Robot.java

```diff
@@ -29,6 +32,8 @@ public class Robot extends LoggedRobot {
     private RobotContainer robotContainer;
     private Command autonomousCommand;
 
+    private final TalonFX motor = new TalonFX(1, "caniboy");
```

Comment:
Find a more professional name for the CANivore. Also, I'd rename the motor to better reflect it's specifically connected to a CANivore.

Reply:
I did this as a temporary experiment, I don't know why I pushed it.

Reply:
No worries. Now you can use interactive rebasing to delete/edit the commit that introduced this change 🙂